infermedica_api/models/red_flag.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+"""
+infermedica_api.models.red_flag
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module contains RedFlag models for data returned from api.
```

Comment:
... contains red flag models ... "RedFlag" is no any special name. Similarly in other doc strings in this file.

Reply:
What do you mean? `RedFlag` is the name of the model that represents a concept of a "red flag", i.e. a general term for an observation that that may be related to potentially life-threatening conditions.

Reply:
It's purely linguistic matter, in my opinion in that sentence you are referring to "red flag models" (more than one model, related to red flags). But at the same time by writing "RedFlag" you refer to just one specific model called "RedFlag" followed by "models" which suggest there is more.
I am also not an expert in English so I may be wrong, if so on other model files we have wrong doc strings (see other model files).

Reply:
Oh, I see what you mean. Sure, I will update that to be consistent with other places. Thanks!